### DISCUSSION TITLE
Is there any way to have rofi focused when you open it instead of hovering over it/clicking on it with the mouse?

### DISCUSSION THREAD

**Discussion by ghost** [JaKooLit/Hyprland-Dots#196] [Q&A]
This is really bugging me out and I have not yet found a solution to this. Every time I open anything related to rofi it does not automatically focus on that window and I have to manually focus on the window.  Is there any way in which rofi is focused whenever it is opened?

**Comment by JaKooLit** [comment]
https://github.com/JaKooLit/Arch-Hyprland/issues/46

somebody opened this on my Arch-Hyprland


however, on my system, I have no issue.

But, at the end of this month, once Im at home, I will check on all my distros.

But so far, I have no issue on arch, nixos, gentoo, and fedora

But I will keep this an open issue as a reminder for mw

**Reply by ghost** [reply]
I am currently using Fedora hyprland. And I have tried add window rules for rofi to work around it but have been unsuccessful.

**Comment by ghost** [comment]
I have found a temporary fix by changing from rofi to wofi(wayland fork) which seems to have fixed the problems.

**Comment by ghost** [comment]
This happens if you have rofi already installed before the install script is used. This causes rofi-Wayland to not be installed.
